include/utils/all.h

```diff
@@ -0,0 +1,35 @@
+/*H**********************************************************************
```

Comment:
Please remove all.h. Most of these are already included in MASTODON and MOOSE and may not be needed. If you need a specific library in Quantification.C, for example, include that in Quantification.h.